lib/engines/content_object_store/app/models/concerns/content_object_store/has_audit_trail.rb

```diff
@@ -3,7 +3,7 @@ module HasAuditTrail
     extend ActiveSupport::Concern
 
     included do
-      has_many :content_block_versions, -> { order(created_at: :asc, id: :asc) }, as: :item
```

Comment:
I noticed that this concern is missing unit tests, but found it really hard to write a useful one... welcome any ideas on it.

Reply:
I guess what you could do is create two versions, one with a `created_at` date of 3 days ago, and one with a `created_at` date of today and confirm that the newer one appears first?